user/test/com/google/gwt/emultest/EmulJava9Suite.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Google Inc.
```

Comment:
gwtproject authors?

Reply:
#9823 needs to be fixed first, else checkstyle will fail these files, in the meantime we've been just marking these as Google.